packages/lib-runtime/rollup.config.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-relative-packages
+import { tsLibConfig, dtsLibConfig } from '../common/rollup-configs';
```

Comment:
The reason we're suppressing ESLint `import/no-relative-packages` rule here is due to default [Rollup config file transpile behavior](https://rollupjs.org/guide/en/#using-untranspiled-config-files).

We can revisit and fix this later on.